<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>502</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <strikeout>false</strikeout>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>UpArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Rigid2D-Test v0.0.3-1</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0,1">
    <item>
     <layout class="QGridLayout" name="gridLayout_4" rowstretch="0,0,0">
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupMass">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Mass</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5" columnstretch="0,0,0">
         <item row="0" column="1">
          <widget class="QLabel" name="lbMassLinear">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>linear</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="lbMassRot">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>rotation</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QDoubleSpinBox" name="spinMassLinear">
           <property name="minimum">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum">
            <double>10.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QDoubleSpinBox" name="spinMassRot">
           <property name="minimum">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum">
            <double>10.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupCoeff">
        <property name="title">
         <string>Coeff</string>
        </property>
        <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
         <item row="2" column="0">
          <widget class="QLabel" name="lbFricD">
           <property name="text">
            <string>Friction-D</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="spinFricS"/>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="spinDumper">
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>10.199999999999999</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lbFricS">
           <property name="text">
            <string>Friction-S</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lbSpring">
           <property name="text">
            <string>Spring</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="spinFricD">
           <property name="minimum">
            <double>-3.000000000000000</double>
           </property>
           <property name="maximum">
            <double>0.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.050000000000000</double>
           </property>
           <property name="value">
            <double>-1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbDumper">
           <property name="text">
            <string>Dumper</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="spinSpring">
           <property name="maximum">
            <double>500.000000000000000</double>
           </property>
           <property name="value">
            <double>180.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QGroupBox" name="groupInit">
        <property name="title">
         <string>Initialize</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0" columnstretch="0,0">
         <item row="1" column="0">
          <widget class="QLabel" name="lbNBox">
           <property name="text">
            <string>Num boxies</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinNBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>32</number>
           </property>
           <property name="value">
            <number>3</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QGroupBox" name="groupSim">
        <property name="title">
         <string>Simulation</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QLabel" name="lbSteps">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Num steps</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinNStep">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>50</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Enviroment</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Air resistance</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="lbX_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>linear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinAirX">
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbY_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>rotation</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinAirY">
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Gravity</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="lbX">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>x</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinGravX">
            <property name="minimum">
             <double>-100.000000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.500000000000000</double>
            </property>
            <property name="value">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbY">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>y</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="spinGravY">
            <property name="minimum">
             <double>-100.000000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.500000000000000</double>
            </property>
            <property name="value">
             <double>-8.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chkVSync">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>VSync</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="resetButton">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>reset scene</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>374</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAaa">
    <property name="title">
     <string>&amp;application</string>
    </property>
    <addaction name="actionQuit_q"/>
   </widget>
   <addaction name="menuAaa"/>
  </widget>
  <action name="actionQuit_q">
   <property name="text">
    <string>quit(&amp;q)</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Quit &lt;span style=&quot; font-weight:600;&quot;&gt;the&lt;/span&gt; application&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>spinSpring</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>changeCoeff()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinDumper</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>changeCoeff()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinFricD</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>changeCoeff()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>26</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinFricS</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>changeCoeff()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinNBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>changeInitial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinGravX</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>changeEnv()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinGravY</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>changeEnv()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinAirX</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>changeEnv()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinAirY</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>changeEnv()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinNStep</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>changeInitial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinMassLinear</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>changeInitial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinMassRot</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>changeInitial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkVSync</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>changeView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeCoeff()</slot>
  <slot>changeInitial()</slot>
  <slot>changeEnv()</slot>
  <slot>changeView()</slot>
 </slots>
</ui>
